src/uk/gov/defra/ffc/DefraUtils.groovy

```diff
@@ -167,32 +166,32 @@ def waitForQualityGateResult(timeoutInMinutes) {
   }
 }
 
-def buildAndPushContainerImage(credentialsId, registry, imageName, tag) {
+def buildAndPushContainerImage(credentialsId, registry, projectName, tag) {
   docker.withRegistry("https://$registry", credentialsId) {
-    sh "docker-compose build --no-cache"
-    sh "docker tag $imageName $registry/$imageName:$tag"
-    sh "docker push $registry/$imageName:$tag"
+    sh "docker-compose -f docker-compose.yaml build --no-cache"
+    sh "docker tag $projectName $registry/$projectName:$tag"
+    sh "docker push $registry/$projectName:$tag"
   }
 }
 
-def deployChart(credentialsId, registry, imageName, tag, extraCommands) {
+def deployChart(credentialsId, registry, projectName, tag, extraCommands) {
```

Comment:
`helmChart` would be more accurate, like `lintHelm`.

Reply:
Changed to `chartName`.